pulp_rpm/app/models/distribution.py

```diff
@@ -316,7 +319,7 @@ class Variant(BaseModel):
         DistributionTree, on_delete=models.CASCADE, related_name='variants'
     )
     repository = models.ForeignKey(
-        Repository, on_delete=models.PROTECT, related_name='+', null=True
+        Repository, on_delete=models.DO_NOTHING, related_name='variants', null=True
```

Comment:
I changed related_name to  `variants` to have a backward relation. I'm not sure why only for this model it was chosen not to have it. If it's a bad idea, let me know.
However it helps to identify orphaned subrepos and make a data migration easy.